sdk/src/ingredient.rs

```diff
@@ -670,7 +693,79 @@ impl Ingredient {
                     ingredient.set_thumbnail(format, image)?;
                 }
                 Err(err) => {
-                    dbg!(&err);
+                    log::warn!("Could not create thumbnail. {err}");
+                }
+            }
+        }
+
+        Ok(ingredient)
+    }
+
+    /// Creates an `Ingredient` from a memory buffer (async version).
+    ///
+    /// This does not set title or hash
+    /// Thumbnail will be set only if one can be retrieved from a previous valid manifest
+    pub async fn from_memory_async(format: &str, buffer: &[u8]) -> Result<Self> {
+        let mut stream = Cursor::new(buffer);
+        Self::from_stream_async(format, &mut stream).await
+    }
+
+    /// Creates an `Ingredient` from a stream (async version).
+    ///
+    /// This does not set title or hash
+    /// Thumbnail will be set only if one can be retrieved from a previous valid manifest
```

Comment:
What does this mean?
>Thumbnail will be set only if one can be retrieved from a previous valid manifest

Do you mean automatically? I currently am setting a thumbnail after creating the `Ingredient` that was captured from the editing tool on import - just wanted to confirm that this workflow is valid.